test/e2e/e2e_test.go

```diff
@@ -45,6 +45,7 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/auth"
 	_ "k8s.io/kubernetes/test/e2e/autoscaling"
 	_ "k8s.io/kubernetes/test/e2e/cloud"
+	_ "k8s.io/kubernetes/test/e2e/cloud/gcp"
```

Comment:
This is already handled by `test/e2e/cloud/imports.go`, don't need to add here

Reply:
done